tidb_read_staleness.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 通过系统变量 `tidb_read_staleness` 读取历史数据
```

Comment:
@en-jin19 Please create a PR to modify the file name. In TiDB documentation, `-` is used to connect words in a file name. DO NOT use **_**. Thanks~